docs/pages/push-notifications/fcm-credentials.mdx

```diff
@@ -79,11 +79,38 @@ Here are the steps to configure a new Google Service Account Key in EAS for send
 </Step>
 
 <Step label="5">
+    Configure the **google-services.json** file in your project. Download it from the Firebase Console and place it at the root of your project directory. If you're using version control, be sure to add this to your ignore file (e.g. **.gitignore**) as it contains sensitive data.
```

Comment:
```suggestion
    Configure the **google-services.json** file in your project. Download it from the Firebase Console and place it at the root of your project directory. If you're using version control, add it to your ignore file (for example, **.gitignore**) as it contains sensitive data.
```

- Suggestion to update the verbiage slightly for improved readability
- Also, let's avoid [Latin abbreviations](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#abbreviations) as per our writing style guide.